Monika After Story/game/script-moods.rpy

```diff
@@ -556,3 +556,72 @@ label mas_mood_yearolder_years:
 label mas_mood_yearolder_leap_today:
     # nothing special occurs here for now
     jump mas_mood_yearolder_bday_true
+
+init 5 python:
+    addEvent(Event(persistent._mas_mood_database,"mas_mood_scared",prompt="scared",category=[store.mas_moods.TYPE_BAD],unlocked=True),eventdb=store.mas_moods.mood_db)
+
+label mas_mood_scared:
+    m 1r "{i}There is nothing to fear, but fear itself.{/i}"
+    m 1k "Ahaha~ I'm kidding."
+    m 3a "Though the reason we become scared, is because we sense any form of danger."
+    m "This can be in a physical or psychological form."
```

Comment:
The sentence could look cleaner but using the word "either" or "both", or a combination of the two.
"This can be in either a physical or a psychological form, sometimes both."